src/Illuminate/Foundation/Testing/RefreshDatabase.php

```diff
@@ -125,6 +134,22 @@ protected function connectionsToTransact()
                             ? $this->connectionsToTransact : [null];
     }
 
+    /**
+     * Checks if the number of transaction starts and ends match at the end of a test.
+     *
+     * @return void
+     */
+    public function withCheckTransactionLevel() {
```

Comment:
IMHO

Given its use, shouldn't this be protected?

Reply:
Thank you, you're right! Fixed both points.